packages/client-sdk-web/test/integration/integration-setup.ts

```diff
@@ -12,13 +14,28 @@ import {CacheClient, TopicClient, Configurations, AuthClient} from '../../src';
 import {ITopicClient} from '@gomomento/sdk-core/dist/src/clients/ITopicClient';
 import {ICacheClient} from '@gomomento/sdk-core/dist/src/clients/ICacheClient';
 
-const credsProvider = CredentialProvider.fromEnvironmentVariable({
-  environmentVariableName: 'TEST_AUTH_TOKEN',
-});
+let _credsProvider: CredentialProvider | undefined = undefined;
+
+function credsProvider(): CredentialProvider {
+  if (_credsProvider === undefined) {
+    if (isLocalhostDevelopmentMode()) {
+      _credsProvider = CredentialProvider.fromEnvironmentVariable({
+        environmentVariableName: 'TEST_AUTH_TOKEN',
+        controlEndpoint: 'https://no-controlplane-requests-allowed:9001',
```

Comment:
So [we're allowed to have one request](https://giphy.com/embed/l2Jea35pzsnoJUBDG) 😆